scripts/create_mailgun_route.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+
+"""Create mailing lists for all top-level projects
```

Comment:
Per IRL convo, let's remove this script entirely and just configure the route(s) manually through the Mailgun web app.